app/src/main/java/com/github/sdp/tarjetakuna/ui/singlecard/SingleCardViewModel.kt

```diff
@@ -47,10 +47,10 @@ class SingleCardViewModel : ViewModel() {
         if (!userDB.isConnected()) {
             return
         }
-        DatabaseSync.sync()
+        //DatabaseSync.sync()

```

Comment:
Can you please provide more explanation on why this is commented (just above it, maybe with a todo)

Reply:
No, because it will be removed before the final merge in main.

Reply:
Ok, but it is also useful for the reviewers to know what's going on.

Reply:
It was a draft PR, not a real PR.

Reply:
this still needs to be removed